autometa/binning/recursive_dbscan.py

```diff
@@ -92,73 +197,72 @@ def recursive_dbscan(table, markers_df, domain, completeness_cutoff,
             columns = ['x,'y','z','coverage','cluster','purity','completeness']
     """
     eps = 0.3
-    step = 0.1
     clustering_rounds = {0: 0}
-    n_clusters = float('inf')
-    best_median = float('-inf')
+    n_clusters = float("inf")
+    best_median = float("-inf")
     best_df = pd.DataFrame()
     while n_clusters > 1:
-        binned_df = run_dbscan(table, eps, method=method)
-        df, metrics_df = add_metrics(
-            df=binned_df, markers_df=markers_df, domain=domain)
-        completeness_filter = metrics_df['completeness'] >= completeness_cutoff
-        purity_filter = metrics_df['purity'] >= purity_cutoff
-        median_completeness = metrics_df[completeness_filter &
-                                         purity_filter]['completeness'].median()
+        binned_df = run_dbscan(table, eps)
+        df, metrics_df = add_metrics(df=binned_df, markers_df=markers_df, domain=domain)
+        completeness_filter = metrics_df["completeness"] >= completeness_cutoff
+        purity_filter = metrics_df["purity"] >= purity_cutoff
+        median_completeness = metrics_df[completeness_filter & purity_filter][
+            "completeness"
+        ].median()
         if median_completeness >= best_median:
             best_median = median_completeness
             best_df = df
         # Count the number of clusters
-        n_clusters = df['cluster'].nunique()
+        n_clusters = df["cluster"].nunique()
         if n_clusters in clustering_rounds:
             clustering_rounds[n_clusters] += 1
         else:
             clustering_rounds[n_clusters] = 1
         # We speed this up if we are getting a lot of tables with the same number of clusters
         if clustering_rounds[n_clusters] > 10:
-            step *= 10
+            eps *= 10
```

Comment:
This changes behavior. Now the step change will depend on the current value of eps. Is this intended? Before we would have `step = 0.1`, and then if we had more than 10 rounds with the same number of clusters, it would change to `1.0`.